site/content/en/docs/Guides/Best Practices/gke.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "GKE Best Practices"
+linkTitle: "GKE"
+date: 2023-05-12T00:00:00Z
+description: "Best practices for running Agones on GKE."
+---
+
+## Overview
+
+On this page, we've collected several GKE best practices.
+
+## Release Channels
+
+### Why?
+
+We recommned using [Release Channels](https://cloud.google.com/kubernetes-engine/docs/concepts/release-channels) for all GKE clusters. Using Release Channels has several advantages:
+* Google automatically manages the version and upgrade cadence for your Kubernetes Control Plane and its nodes.
+* Clusters on a Release Channel are allowed to use the `No minor upgrades` and `No minor or node upgrades` [scope of maintenance exclusions](https://cloud.google.com/kubernetes-engine/docs/concepts/maintenance-windows-and-exclusions#limitations-maint-exclusions) - in other words, enrolling a cluster in a Release Channel gives you _more control_ over node upgrades.
+* Clusters enrolled in `rapid` channel have access to the newest Kubernetes version first. Agones strives to [support the newest release in `rapid` channel]({{< relref "Installation#agones-and-kubernetes-supported-versions" >}}) to allow you to test the newest Kubernetes soon after it's available in GKE.
+
+{{< alert title="Note" color="info" >}}
+GKE Autopilot clusters must be on Release Channels.
+{{< /alert >}}
+
+### What channel should I use?
+
+We recommend the `regular` channel, which offers a balance between stability and freshness. See [this guide](https://cloud.google.com/kubernetes-engine/docs/concepts/release-channels#what_channel_should_i_use) for more discussion.
+
+If you need to disallow minor version upgrades for more than 6 months, consider choosing the freshest Kubernetes version possible: Choosing the freshest version on `rapid` or `regular` will extend the amount of time before your cluster reaches [end of life](https://cloud.google.com/kubernetes-engine/docs/release-schedule#schedule-for-release-channels).
+
+### What versions are available on a given channel?
+
+You can query the versions available across different channels using `gcloud` using:
+
+```
+gcloud container get-server-config \
+  --region=[COMPUTE_REGION] \
+  --flatten="channels" \
+  --format="yaml(channels)"
+```
+Replace the following:
+
+* **COMPUTE_REGION**: the
+[Google Cloud region](https://cloud.google.com/compute/docs/regions-zones#available)
+for the cluster.
```

Comment:
"for the cluster" isn't quite accurate, since presumably you are doing this before creating a cluster. What about "where you will create the cluster" or something like that?